gradle.properties

```
@@ -48,7 +48,7 @@ stdlibAuthVersion=2.7.0
 stdlibJwtVersion=2.7.0
 stdlibOAuth2Version=2.7.0
 
-stdlibHttpVersion=2.7.1
+stdlibHttpVersion=2.7.3-20231012-153400-1cc836c
```

Need to update the version after http release
```suggestion
stdlibHttpVersion=2.7.3
```